server/job/blacklistedTagsProcessor.ts

```diff
@@ -121,6 +121,8 @@ class BlacklistedTagProcessor implements RunnableScanner<StatusBase> {
               page,
               sortBy,
               keywords: tag,
+              includeAdult: true,
+              includeVideo: true,
```

Comment:
> 'Video' content from tmdb would not be considered in the blacklisted tags job, but would be shown in the search results. This resulted in a small subset of undesired movies that couldn't be auto-blacklisted based on their tags

Adding `includeVideo` makes sense as you mentioned that `video` content wasn't being returned by blacklist job endpoints but was appearing in search results, so videos need to be explicitly included for proper blacklisting.

However, why is `includeAdult: true` needed here? Search endpoints already filter out adult content by default (`includeAdult = false`), so users never see it anyway. Why blacklist content that's already filtered on an api level and never reaches users?